src/lib/server/migrations.ts

```diff
@@ -436,6 +440,42 @@ const migrations = [
 		run: async () => {
 			await collections.pictures.dropIndex('productId_1').catch(console.error);
 		}
+	},
+	{
+		_id: new ObjectId('669a90d18bc5aaf40c863b63'),
+		name: 'Adding actionSettings nostr to products',
+		run: async (session: ClientSession) => {
+			await collections.products.updateMany(
+				{},
+				{
+					$set: {
+						'actionSettings.nostr': {
+							visible: true,
+							canBeAddedToBasket: true
+						}
+					}
+				},
+				{ session }
+			);
+		}
+	},
+	{
+		_id: new ObjectId('669e255c5cf12669421d9a2a'),
+		name: 'Adding actionSettings nostr to runtimConfig',
+		run: async (session: ClientSession) => {
+			await collections.runtimeConfig.updateOne(
+				{ _id: 'productActionSettings' },
+				{
+					$set: {
+						'data.nostr': {
+							visible: true,
+							canBeAddedToBasket: true
+						}
+					}
+				},
+				{ session }
+			);
+		}
 	}
```

Comment:
No need to create two migrations in one PR, you can merge it with the previous one